src/ripple/protocol/impl/LedgerFormats.cpp

```diff
@@ -115,15 +117,16 @@ LedgerFormats::LedgerFormats()
             {sfAccount,              soeREQUIRED},
             {sfDestination,          soeREQUIRED},
             {sfAmount,               soeREQUIRED},
+            {sfTransferRate,         soeREQUIRED},
```

Comment:
why is this required instead of being optional or default?

Reply:
also curious would adding a new `soeREQUIRED` field cause issues for existing objects?